test/unit/api/python/test_finite_field.py

```diff
@@ -53,3 +53,43 @@ def test_basic_finite_field(solver):
     solver.assertFormula(bIsTwo)
     r = solver.checkSat()
     assert not r.isSat()
+
+def test_basic_finite_field_base(solver):
```

Comment:
This test should also be added for C++.

Reply:
I can't find `test_basic_finite_field` like cpp test, where should I add this?

Reply:
Right, you added these two tests, that's why they don't exist for C++ yet. They should be added to https://github.com/cvc5/cvc5/blob/main/test/unit/api/cpp/solver_black.cpp

Reply:
The first one was not mine, but ok, will be done.

Reply:
Ah right, thanks for adding it anyway!